docs/Clearwater_Configuration_Options_Reference.md

```diff
@@ -221,6 +223,23 @@ This section describes settings that may vary between systems in the same deploy
 * `authentication` - by default, Clearwater performs authentication challenges (SIP Digest or IMS AKA depending on HSS configuration). When this is set to 'Y', it simply accepts all REGISTERs - obviously this is very insecure and should not be used in production.
 * `num_http_threads` (Homestead) - determines the number of HTTP worker threads that will be used to process requests. Defaults to 50 times the number of CPU cores on the system.
 
+## DNS Config
+
+This section describes the static DNS config which can be used by Chronos. This is set in `/etc/clearwater/dns_config`. Currently, the only supported record type is CNAME. The file has the format:
```

Comment:
I'd don't want this to sound too Chronos specific. How about: This section describes the static DNS config which can be used to override DNS results. This is set in `etc/clearwater/dns_config`. Currently, the only supported record type is CNAME, and the only component that uses this is Chronos.